README.md

```diff
@@ -88,6 +88,16 @@ With that in mind, the steps for adding a new API are:
 4. Wait for CI to pass, fixing errors.
 5. Wait for a merge!
 
+### Test before you commit
+
+We have two automated tests running on [Travis](https://travis-ci.org/rust-lang/libc):
+
+1. `libc-test`
+  - `cd libc-test && cargo clean && cargo run`
```

Comment:
`cargo clean` is for making sure `all.c` is regenerated according to the latest code of the `libc` crate.

```
$ cd libc-test; cargo build
$ echo 'extern { pub fn xyzzy() -> ::c_int; }' >> ../src/unix/mod.rs
$ cargo run
   Compiling libc v0.2.9 (file:///tmp/libc/libc-test)
   Compiling libc-test v0.1.0 (file:///tmp/libc/libc-test)
     Running `target/debug/libc-test`
RUNNING ALL TESTS
PASSED 6086 tests
$ cargo clean; cargo run                                                                                                                                                                                                                 (git)-master 
   Compiling winapi v0.2.6
   Compiling winapi-build v0.1.1
   Compiling gcc v0.3.25
   Compiling libc v0.2.8
   Compiling unicode-xid v0.0.3
   Compiling bitflags v0.3.3
   Compiling rustc-serialize v0.3.18
   Compiling libc v0.1.12
   Compiling libc v0.2.9 (file:///tmp/libc/libc-test)
   Compiling kernel32-sys v0.2.1
   Compiling term v0.2.14
   Compiling log v0.3.5
   Compiling syntex_syntax v0.19.1
   Compiling ctest v0.1.0 (https://github.com/alexcrichton/ctest#50ac771a)
   Compiling libc-test v0.1.0 (file:///tmp/libc/libc-test)
failed to run custom build command for `libc-test v0.1.0 (file:///tmp/libc/libc-test)`
Process didn't exit successfully: `/tmp/libc/libc-test/target/debug/build/libc-test-475f1c4bf6cbc262/build-script-build` (exit code: 101)
...snip...
--- stderr
/tmp/libc/libc-test/target/debug/build/libc-test-475f1c4bf6cbc262/out/all.c: In function ‘__test_fn_xyzzy’:
/tmp/libc/libc-test/target/debug/build/libc-test-475f1c4bf6cbc262/out/all.c:12758:24: error: ‘xyzzy’ undeclared (first use in this function)
                 return xyzzy;
                        ^
/tmp/libc/libc-test/target/debug/build/libc-test-475f1c4bf6cbc262/out/all.c:12758:24: note: each undeclared identifier is reported only once for each function it appears in
/tmp/libc/libc-test/target/debug/build/libc-test-475f1c4bf6cbc262/out/all.c:12759:13: error: control reaches end of non-void function [-Werror=return-type]
             }
             ^
...snip...
```